appbuilder/services/livesync/ios-livesync-service.ts

```diff
@@ -0,0 +1,74 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import iOSProxyServices = require("../../../mobile/ios/device/ios-proxy-services");
+import * as path from "path";
+import * as shell from "shelljs";
+let osenv = require("osenv");
+
+export class IOSLiveSyncService implements IPlatformLiveSyncService {
+	private get $project(): any {
+		return this.$injector.resolve("project");
+	}
+
+	private get $projectConstants(): IProjectConstants {
```

Comment:
making this a `get` property should not be necessary as this class cannot cause a circular dependency - is there any particular reason for doing so?